_test/test_experiment/test_experiment_cnstrct_and_properties.m

```diff
@@ -75,6 +76,7 @@ function test_nontrivial_runid_map(~)
                 'Samples in experiment are defined but their lattice is undefined');
             assertEqual(b, ...
                 'HORACE:Experiment:lattice_undefined');
+            lastwarn('');
             
```

Comment:
Minor: you do not need to do `lastwarn`  here as next warning will superseed it. 
also 
```
clob = set_temporary_warning('off','whatever was issued for tests') 
```
should be used to reduce noise while running tests (the warning will be issued anyway, but would not be displayed)